data.py

```diff
@@ -1,27 +1,48 @@
 import requests
-
-
-def get_data_from_api():
-    url = "https://randomuser.me/api/?results=10" #Przykladowe API
-    response = requests.get(url)
-    api_data = response.json()
-    return api_data["results"]
-
-
-def display_data(api_data):
-    for user in api_data:
-        print("First Name:", user["name"]["first"])
-        print("Last Name:", user["name"]["last"])
-        print("Email:", user["email"])
-        print("City:", user["location"]["city"])
-        print("Country:", user["location"]["country"])
-        print(20 * "-")
-
-
-def main():
-    api_data = get_data_from_api()
-    display_data(api_data)
+from dataclasses import dataclass
+
+
+@dataclass
+class UserData:
+    first_name: str
+    last_name: str
+    email: str
+    city: str
+    country: str
+
+    def __str__(self):
+        return f"Name: {self.first_name} {self.last_name}\nEmail: {self.email}\nLocation: {self.city}, {self.country}\n"
+
+
+class APIClient:
+    def __init__(self):
+        self.base_url = "https://randomuser.me/api/"
+
+    def get_data_from_api(self, results=10):
```

Comment:
let's typehint throughout this file :+1:

Reply:
you should avoid hardcoded values like this `10`, better to 
- get the value from entrypoint as attr in CLI
- set const variable on top of this file